Specs/Number/Japanese/NumberModel.json

```diff
@@ -1447,6 +1544,7096 @@
   },
   {
     "Input": "私の家は四川にあります",
+    "Comment": "PendingValidation",
+    "Results": []
+  },
+  {
+    "Input": "答えは１９２です。",
+    "IgnoreResolution": true,
```

Comment:
When revising/extending coverage in future PRs, please try to remove this IgnoreResolution from as many cases as possible and see if parsing behaves correctly. 
If parsing results are correct, keep resolution check enabled. For the new spec cases where parsing is incomplete and doesn't work, please re-add the IgnoreResolution tag.